src/user/tipsandtricks.rst

```diff
@@ -28,20 +28,20 @@ Why does that happen?
 
 The ``conda-forge`` and ``defaults`` are not 100% compatible.
 In the example above it is known that ``defaults`` uses ``icu 54.*`` while ``conda-forge`` relies on ``icu 56.*``,
-that mismatch can lead to errors when the install environment is mixing packages from multiple channels.
+this mismatch can lead to errors when the install environment is mixing packages from multiple channels.
```

Comment:
Looks like this sentence has been a run-on for a while. Feel free to keep your change, but replace that comma with a semi-colon or start a new sentence at "This mismatch..."